cmd/execinquery/main.go

```diff
@@ -1,9 +1,8 @@
 package main
 
 import (
-	"golang.org/x/tools/go/analysis/unitchecker"
```

Comment:
My package is separated from other packages and declarations. Therefore, this edit is not necessary.